docs/source/notes/cpu_threading_torchscript_inference.rst

```diff
@@ -0,0 +1,126 @@
+.. _cpu-threading-torchscript-inference:
+
+CPU threading and Torchscript inference
+=================================================
+
+PyTorch allows using multiple CPU threads during TorchScript model inference.
+The following figure shows different levels of parallelism one would find in a
+typical application:
+
+.. image:: cpu_threading_torchscript_inference.png
```

Comment:
Can we make this a vector graphic? 78kB tucked to the repo history forever is not a idea 😕